src/main/java/com/google/sps/data/backgroundTaskManagerInterface.java

```diff
@@ -0,0 +1,20 @@
+package com.google.sps.data;
+import java.util.Optional;
+import java.util.List;
+
+/** Interface for the backgroundTaskManager class. */
+public interface backgroundTaskManagerInterface { 
+/**
+   * Checks VM status to make sure they are healthy. 
+   */
+    void checkVM();
+/**
+   * Removes attendee if they haven't been polled in 2 minutes.
+   * Nothing associated with if the attendee is the controller.  
+   */
+    void removeAttendee(); 
+ /**
+   * Finds or creates a replacement VM after a VM's status is checked.
+   */
+   void replaceVM();
```

Comment:
void replaceFaultyInstances();

Will simply list all the sessions and for each session, if its instance is *not* running, it will find a replacement or create a new one.